.eslintignore

```diff
@@ -0,0 +1 @@
+/src/external/**/*.js
```

Comment:
Just general question: why is it needed?
I don't see such a folder in repository.

Reply:
It was there in the previous Gruntfile. Happy to remove it if it's not needed.

Reply:
I don't have this knowledge. We need a confirmation from the Elders ;)

Reply:
If the folder doesn't exist, then there's no reason to keep it. I think it was for the reqwest we were using before or something like that

Reply:
yes we can remove this